shared/src/commonMain/kotlin/io/customer/shared/common/TypeAliases.kt

```diff
@@ -0,0 +1,24 @@
+package io.customer.shared.common
+
+typealias CustomAttributesCompat = Map<String, Any>
```

Comment:
```suggestion
typealias CustomAttributes = Map<String, Any>
```

I am assuming `Compat` was added since that's common naming in Android. 

I don't see value in adding it here. I suggest we just keep internal types called what we refer to them internally in the company as.

Reply:
`CustomAttributes` is already available in this file. The `Compat` is temporary and will be removed soon. Please read comments below `CustomAttributesCompat` to understand the reason of keeping these two temporarily.